src/server/lib/okta/api/headers.ts

```diff
@@ -1,8 +1,19 @@
 import { getConfiguration } from '@/server/lib/getConfiguration';
 
-export const defaultHeaders = {
-	Accept: 'application/json',
-	'Content-Type': 'application/json',
+export const defaultHeaders = (ip?: string) => {
```

Comment:
This is the main change for the Okta API side, to convert `defaultHeaders` to a function that takes the `ip`